### PR TITLE
Replace dummy data

### PR DESCRIPTION
**Pull request by ninalouw**
@thaisamarinho 
@mrreubenjones 
@jeffcstock 
@whilestevego 

I replaced the dummy data in the lost pets scroll screen next to the map. I created a file called home.js, I made ajax requests to get the JSON data, and I copied the relevant HTML from home/index.html.erb into my js functions. I also changed the image_uploader to use fog only if in production.

### CODE REVIEW

**Comment by whilestevego** [comment]
@ninalouw good description. I clearly know what you did. However, when tagging someone for review. Only tag one person at a time. It's ambiguous who should review at the moment.

File: app/assets/javascripts/home.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+$(document).ready(function(){
+ var petList = $("#recent-pet-list");
+ getPets();
+ });
+
+ function getPets(){
+   $.ajax({
+     url:`api/v1/pets`,
+     success: function(pets){
+       console.log(pets[0])
```
Comment (thaisamarinho) [inline app/assets/javascripts/home.js]: Do you still need to console.log[0] at line 11 over there?

File: app/assets/javascripts/home.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }).join("");
+
+  petList.html(petsHTML);
+}
+
+function PetListItem (pet) {
+  return `
+  <div class="main-list">
+    <div class="col-md-3 col-xs-6">
+      <img class='img-circle' src="${pet.image}" alt="" />
```
Comment (thaisamarinho) [inline app/assets/javascripts/home.js:35]: have you check this code to see if works with a pet with more than one picture on its profile? the fact that there is pet.image makes me think about what if this pet has 2 or 3 images...

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -8,9 +8,7 @@
               <img src="/assets/panel-image-1.png" alt="" />
             </a>
           </div>
-
           <div class="col-md-5">
-
           <% if user_signed_in? %>
```
Comment (thaisamarinho) [inline app/views/home/index.html.erb]: can you indent this code? Maybe is already indented, but this file is so big that i think i got lost.

**Review by thaisamarinho — COMMENTED** [review]
Nina, Great job! Awesome work over here. I made some small comments. Thanks!!

File: app/assets/javascripts/home.js (hunk 1 above)
Comment (ninalouw) [reply app/assets/javascripts/home.js]: Deleted :)

File: app/assets/javascripts/home.js (hunk 2 above)
Comment (ninalouw) [reply app/assets/javascripts/home.js:35]: I'll look into that now, good point!

Comment (ninalouw) [reply app/assets/javascripts/home.js:35]: It works! In the report it shows 3 images, and in the Lost pet scrollbar it just shows the first one :)

File: app/views/home/index.html.erb (hunk 3 above)
Comment (ninalouw) [reply app/views/home/index.html.erb]: Yes I dont really understand the indentation here either, I'll make some changes and then I'll push it all up again :)

**Comment by ninalouw** [comment]
@whilestevego , @thaisamarinho  has already reviewed this and it seems to be working :)

**Review by whilestevego — CHANGES_REQUESTED** [review]
You have a `.DS_Store` file in here. It should not be there. Can you add ignore rules for MacOS? Go to https://www.gitignore.io/ and remove this

File: app/uploaders/image_uploader.rb [hunk 4]
```diff
@@ -1,11 +1,54 @@
...
+  # end
+
+  # Process files as they are uploaded:
+  # process scale: [200, 300]
+  #
+  # def scale(width, height)
+  #   # do something
+  # end
+
+  # Create different versions of your uploaded files:
```
Comment (whilestevego) [inline app/uploaders/image_uploader.rb]: Can you delete these comments?

File: app/uploaders/image_uploader.rb [hunk 5]
```diff
@@ -17,4 +60,29 @@ def store_dir
...
+  # For images you might use something like this:
+  # def extension_whitelist
+  #   %w(jpg jpeg gif png)
+  # end
+
+  # Override the filename of the uploaded files:
+  # Avoid using model.id or version_name here, see uploader/store.rb for details.
+  # def filename
+  #   "something.jpg" if original_filename
+  # end
```
Comment (whilestevego) [inline app/uploaders/image_uploader.rb]: Delete these too.

File: app/assets/javascripts/home.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
+
+$(document).ready(function(){
+ var petList = $("#recent-pet-list");
+ getPets();
+ });
```
Comment (whilestevego) [inline app/assets/javascripts/home.js:6]: Indenting levels are off here. 📇

File: app/assets/javascripts/home.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
+
+$(document).ready(function(){
```
Comment (whilestevego) [inline app/assets/javascripts/home.js]: Can you add a space after `()`?

File: app/uploaders/image_uploader.rb [hunk 8]
```diff
@@ -1,11 +1,54 @@
...
+
+  # Process files as they are uploaded:
+  # process scale: [200, 300]
+  #
+  # def scale(width, height)
+  #   # do something
+  # end
+
+  # Create different versions of your uploaded files:
+
```
Comment (whilestevego) [inline app/uploaders/image_uploader.rb]: Delete these comments to?

**Review by whilestevego — CHANGES_REQUESTED** [review]
Few changes to do

**Review by whilestevego — APPROVED** [review]
Merging this

**Pull request merged**